FrameworkInternals/quasar_busybox.py

```diff
@@ -0,0 +1,45 @@
+'''
+quasar_busybox.py
```

Comment:
Why call this file 'busybox.py' ? I would have thought this python function is a candidate for addition to quasar_basic_utils.py or maybe manage_files.py

Reply:
The inspiration for the name came from the BusyBox:
https://en.wikipedia.org/wiki/BusyBox
I believe this is a common name in the software engineering world, but we can discuss it.

Why not quasar_basic_utils or manage_files? I thought that neither of them actually delivers(-ed) functions which could be independent functions callable directly from the shell (thus, available directly to cmake). I preferred to keep separation between what is internal to quasar's python-based scripting (i.e. typically called via quasar.py) and what is external; the latter was not represented yet so that's why I created a new file. 

So keeping in mind the separation, I think that adding a new file was justified.

Let me know your viewpoint.

Reply:
busybox sounds unix/linux-y to me, so, unlikely to be a place you'd look into if you're a windows developer poking about in quasar. Windows == first class quasar citizen and all that.

Reply:
Propose move to manage_files.py

Reply:
as per Windows comment... and 1st/2nd class citizen... well.. not sure about it, to be checked.

But for the organization reasons it remains a valid question. @parasxos , what is your viewpoint on it, since you already reviewed this part?

Reply:
To my mind a new file is justified if it has a clear separate scope different than the existing ones.

The name of the library could be descriptive of its scope, as it is the developers that are going to use it and our lives to be made easier. 

Honestly, I didn't think of Ben's concern when reviewing. I think that a more descriptive OS-agnostic name could be used to ease that concern e.g. why not consider manage_files_external.py or utils_external.py or something similar implying the different scope?

Would that be enough?